src/ui/Header/Header.jsx

```diff
@@ -200,6 +200,32 @@ const Main = ({
     }
   };
 
+  // Listens for escape keydown event
+  React.useEffect(() => {
+    const escKeyPressed = (e) => {
+      if (e.key === 'Escape') {
+        // menuOnClickOutside();
```

Comment:
useEffect to listen for escape keydown
FYI: By calling the menuOnClickOutside() got a warning and didn't want to change the function because it wasn't created by me. So I took the function content and now we have some duplicate code.